<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>ed_iva_ret_cta</class>
 <widget class="QDialog" name="ed_iva_ret_cta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asociar IVA y retención a cuentas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="cuentalabel">
     <property name="maximumSize">
      <size>
       <width>740</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>cuenta</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descriplabel">
     <property name="maximumSize">
      <size>
       <width>740</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>descripción</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="ivaasocgroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>IVA asociado</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Cuenta de IVA:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cuentaIVAlineEdit">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ctaIVApushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>25</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="keme.qrc">
              <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="descripctaIVAlabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>315</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>- -</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Tipo::</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tipoIVAcomboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>276</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cuenta base IVA:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ctaBaseIVAlineEdit">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ctaBaseIVApushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>25</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="keme.qrc">
              <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="descripctaBaseIVAlabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>315</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>- -</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="retgroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>336</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Retención asociada</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>28</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Cuenta ret.:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cuentata_ret_asiglineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="ctaretpushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>25</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="keme.qrc">
              <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="cta_ret_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>195</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>- -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="es_ret_asigcheckBox">
            <property name="text">
             <string>Arrendamiento</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Op:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tipo_oper_ret_asigcomboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>282</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>%:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="porc_ret_asig_lineEdit">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Concepto asociado en diario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="conceptodiariolineEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/check2.png</normaloff>:/iconos/graf/check2.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ivaasocgroupBox</tabstop>
  <tabstop>cuentaIVAlineEdit</tabstop>
  <tabstop>ctaIVApushButton</tabstop>
  <tabstop>tipoIVAcomboBox</tabstop>
  <tabstop>ctaBaseIVAlineEdit</tabstop>
  <tabstop>ctaBaseIVApushButton</tabstop>
  <tabstop>retgroupBox</tabstop>
  <tabstop>cuentata_ret_asiglineEdit</tabstop>
  <tabstop>ctaretpushButton</tabstop>
  <tabstop>es_ret_asigcheckBox</tabstop>
  <tabstop>tipo_oper_ret_asigcomboBox</tabstop>
  <tabstop>porc_ret_asig_lineEdit</tabstop>
  <tabstop>conceptodiariolineEdit</tabstop>
  <tabstop>aceptarpushButton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ed_iva_ret_cta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
